### ISSUE TITLE
Strange behaviour when repeating RotateAction with a delta

### ISSUE THREAD

**Issue by mattiashagstrand** [JohnSundell/ImagineEngine#81]
If you try to repeat a RotateAction with a delta the rotation is reset to the original value each time the rotation action repeats. I expected the rotation to continue at a constant rate until cancelled.

Example code:

`actor.repeat(RotateAction(delta: 3.14, duration: 1))`

I think all that is needed is to add this function to MetricAction:

```
public override func start(for object: Object) {
    startMetric = nil
}
```

This is consistent with how MoveAction is implemented.

**Comment by JohnSundell** [comment]
Hi @mattiashagstrand 👋

That's definitely a bug! Do you want to submit the fix you suggested as a PR? 🙂

**Comment by mattiashagstrand** [comment]
Absolutely, I will create a PR!

**Comment by JohnSundell** [comment]
Awesome 🎉

**Comment by mattiashagstrand** [comment]
Hmm, looks like I don't have permissions to create a new branch from master. How can I fix that?

**Comment by JohnSundell** [comment]
@mattiashagstrand Fork the repo, make a new branch on your fork, then submit it as a PR 👍

**Comment by JohnSundell** [comment]
Fixed by @mattiashagstrand in #84 🎉
